test-case/check-signal-stop-start.sh

```diff
@@ -16,9 +16,10 @@
 ##    no errors for aplay/arecord
 ##
 
-source $(dirname ${BASH_SOURCE[0]})/../case-lib/lib.sh
+# shellcheck source=caselib/lib.sh
+source "$(dirname "${BASH_SOURCE[0]}")"/../case-lib/lib.sh
 
-OPT_NAME['t']='tplg'     OPT_DESC['t']='tplg file, default value is env TPLG: $TPLG'
+OPT_NAME['t']='tplg'     OPT_DESC['t']="tplg file, default value is env TPLG: $TPLG"
```

Comment:
`-h` already shows the default value (please try changed code) so you can drop the last `$TPLG` bit.